src/modules/order/order.controller.ts

```diff
@@ -45,7 +45,8 @@ export class OrderController {
   @Put(':id/book')
   async bookOrder(
     @Param('id') id: string,
-    @Body() { driverId, transportIds }: { driverId: number, transportIds?: number[] },
+    @Body()
+    { driverId, transportIds }: { driverId: number; transportIds?: number[] },
```

Comment:
May transportIds? be nullable in any case?
Maybe better add Dto class with validations?